sickbeard/providers/tvchaosuk.py

```diff
@@ -102,7 +100,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
             for search_string in search_strings[mode]:
 
                 if mode == 'Season':
-                    search_string = re.sub(ur'(.*)Season', ur'\1Series', search_string)
+                    search_string = re.sub(ur'(.*)S0?', ur'\1Series', search_string)
```

Comment:
There are no Season 6, just Series 6.
They are all like that or at least 90%, making the 10% despicable